package.json

```diff
@@ -58,7 +58,7 @@
     "@embroider/test-setup": "^0.48.1",
     "@glimmer/component": "^1.0.4",
     "@glimmer/tracking": "^1.0.4",
-    "@lblod/ember-rdfa-editor": ">=0.57.0",
+    "@lblod/ember-rdfa-editor": "^0.60.1",
```

Comment:
I would still like to use the same modifier as before, so using `>=`, mainly because this is the way most plugins are configured by now, but also so that whenever you do an `npm install` you get the very latest version of the editor to test the plugin against. I know that this feels a bit obfuscated, but remembering to manually set the latest editor version before developing a new feature also feels obfuscated.

Reply:
I also don't want to run into situations where setting the editor version to 0.61.0 on the frontend in the future causes problems because this plugin requires an older version of the editor.

Reply:
Ah okay, I understand, I will update it to >=0.60.0